app/assets/stylesheets/sass/views/attendances/_new.css.scss

```diff
@@ -0,0 +1,109 @@
+body {
+  .container {
+    header {
+      .logo {
+        background-color:$very-dark-cyan;
+        border-radius: 2rem;
+        padding: 1rem;
+        text-align: center;
+        width: 50rem;
+        
+        h1 {
+          color: white;
+          font-size: 6rem;
+          font-weight: 400;
+        }
+      }
+    }
+
+    main {
+      column-gap: 2rem;
+      display: flex;
+      justify-content: center;
+      width: 80%;
+
+      div.check {
+        div.numeric-pad {
+          background-color: $very-dark-cyan;
+          border-radius: 1rem;
+          display: flex;
+          flex-direction: column;
+          gap: 8px;
+          height: 30rem;
+          padding: 1rem;
+          width: 25rem;
+          
+          div.screen {
+            input[type=password] {
+              background: $normal-cyan;
+              border-radius: 1.3rem;
+              color: white;
+              font-size: 4rem;
+              padding-right: 0.4rem;
+              text-align: right;
+              width: 100%;
+            }
+          }
+
+          div.numbers-button {
+            column-gap: 0.8rem;
+            display: grid;
+            grid-template-columns: 1fr 1fr 1fr;
+            height: 100%;
+            padding: 0;
+            row-gap: 0.8rem;
+            width: 100%;
+
+            input:nth-child(10) { 
+              grid-column: 2/3;
+              grid-row: 4/5;
+            }
+
+            input:nth-child(11) { 
+              grid-column: 3/4;
+              grid-row: 4/5;
+            }
+
+            input[type=button] {
+              background-color:$normal-cyan;
+              border-color: $normal-cyan;
+              border-radius: 1rem;
+              border-width: 2px;
+              box-shadow: 1px 1px 1px 1px black;
+              color: white;
+              font-size: 2rem;
+              height: 100%;
+              width: 100%;
+            }
+          }
+        }
+
+        div.checks-buttons {
+          display: flex;
+          justify-content: space-around;
+          margin-top: 1rem;
+        }
+      }
+
+      div.right-side {
+        @include center-flex-column;
+
+        div.clock {
+          @include center-flex-column;
+          background-color: $very-dark-cyan;
+          border-radius: 1rem;
+          color: white;
+          font-size: 5rem;
+          gap: 2rem;
+          height: 30rem;
+          text-align: center;
+          width: 45rem;
+        }
+
+        button {
+          margin-top: 1rem;
+        }
+      }
+    }
+  }
+}
```

Comment:
Check the indentation of this file, the recommended spacing is two.

Reply:
@YilKanda Indeed the actual indentation is two spacing